docs/imviz/displayimages.rst

```diff
@@ -0,0 +1,85 @@
+.. _imviz-display-images:
+
+*****************
+Displaying Images
+*****************
+
+Imviz uses image viewers to visualize data from supported formats.
+
+Selecting Data Set
+==================
+
+.. seealso::
+
+    :ref:`Selecting Data Set<cubeviz-selecting-data>`
+        Documentation on selecting data sets in the Jdaviz viewers.
+
+
+Adding New Viewers
+==================
+
+In the toolbar towards the top of the UI, there is a :guilabel:`picture with +` icon. By
```

Comment:
Any chance you can just include a small image of the icon here? "picture with +" is very unwieldy compared to just showing the icon.